src/draw/lv_draw_rect.h

```diff
@@ -128,6 +136,14 @@ void lv_draw_fill_dsc_init(lv_draw_fill_dsc_t * dsc);
  */
 lv_draw_fill_dsc_t * lv_draw_task_get_fill_dsc(lv_draw_task_t * task);
 
+/**
+ * Fill an area
+ * @param layer         pointer to a layer
+ * @param dsc           pointer to an initialized draw descriptor variable
+ * @param coords        the coordinates of the rectangle
+ */
+void lv_draw_fill(lv_layer_t * layer, const lv_draw_fill_dsc_t * dsc, const lv_area_t * coords);
```

Comment:
Out of question on this most of the draw descriptors are what hold the coordinates and don't get passed in a parameter to the function that does the drawing. Is there a reason for the deviation with this function/descriptor?